contracts/staking/ExampleRewardCalculator.sol

```diff
@@ -21,11 +21,10 @@ contract ExampleRewardCalculator is IRewardCalculator {
      */
     function calculateReward(
         uint256 stakeAmount,
-        uint64 startTime,
-        uint64 endTime,
+        uint64 uptime,
         uint256, // initialSupply
         uint256 // endSupply
     ) external view returns (uint256) {
-        return (stakeAmount * rewardBasisPoints * (endTime - startTime)) / SECONDS_IN_YEAR / 1000;
+        return (stakeAmount * rewardBasisPoints * uptime) / SECONDS_IN_YEAR / 1000;
```

Comment:
I don't think we want this change in the example. On the p-chain today, rewards are given for the whole staking period as long as an uptime threshold of 80% has been reached. So the calculator should give the full reward. The contract itself can figure out if the validator gets all or none of the rewards calculated.